.fixtures.yml

```diff
@@ -1,8 +1,8 @@
 fixtures:
-  forge_modules:
+  repositories:
```

Comment:
why the switch from forge?

Reply:
I think it would be good to reduce external dependencies, we rely on github anyhow, so why not load it from there. (Git is also better cached across local nodes)

Reply:
that sounds like a good idea